src/components/Details/DetailsComponent.tsx

```diff
@@ -20,8 +28,11 @@ export const DetailsComponent = () => {
   const { nodeLabels, nodeLimit, queryHistory, isPhysicsEnabled } =
     useSelector(selectOptions);
 
+  const [editField, setEditField] = useState<null | string>(null);
+  const [editValue, setEditValue] = useState<null | string>(null);
+
   let hasSelected = false;
-  let selectedType = null;
+  let selectedType: null | undefined = null;
```

Comment:
`null | undefined` does not seem like a good type. It should probably just be type `any`.